docs/feature/1665-custom-sharding.md

```diff
@@ -220,6 +220,109 @@ You can now start a flank test run. With the updated config there will still be
     * `debug-2.apk` with 2 shards
     * `debug-3.apk` with 1 shard
 
+## iOS
+
+iOS custom sharding works exactly the same as Android (dump shards, modify, add a path to JSON file, etc) the same with small exceptions:
+
+* there is no `additional-app-test-apks` feature for iOS
+* every shard in iOS is a separate matrix (FTL limitations)
+* dump shard JSON is different for xctestrun with test plans and not, therefore there is different custom sharding JSON
+  structure for both versions.
+
+#### Custom sharding JSON for xctestrun without Test Plans (example)
+
+```json
+[
+  {
+    "ExampleSwiftTests": [
+      "ExampleSwiftTests/test1",
+      "ExampleSwiftTests/test2",
+      "ExampleSwiftTests/test3",
+      "ExampleSwiftTests/test4",
+      "ExampleSwiftTests/test5"
+    ]
+  },
+  {
+    "ExampleSwiftTests": [
+      "ExampleSwiftTests/test6",
+      "ExampleSwiftTests/test7",
+      "ExampleSwiftTests/test8",
+      "ExampleSwiftTests/test9"
+    ]
+  },
+  {
+    "ExampleSwiftTests": [
+      "ExampleSwiftTests/test10",
+      "ExampleSwiftTests/test11",
+      "ExampleSwiftTests/test12",
+      "ExampleSwiftTests/test13"
+    ]
+  },
+  {
+    "ExampleSwiftTests": [
+      "ExampleSwiftTests/test14",
+      "ExampleSwiftTests/test15",
+      "ExampleSwiftTests/test16",
+      "ExampleSwiftTests/test17"
```

Comment:
Can we change test classes names from `ExampleSwiftTests` to `ExampleSwiftTestsClass` or something similar?